scale-encode/src/lib.rs

```diff
@@ -313,3 +307,16 @@ mod test {
 ///   behaviour and provide your own trait bounds instead using this option.
 #[cfg(feature = "derive")]
 pub use scale_encode_derive::EncodeAsType;
+
+#[cfg(test)]
```

Comment:
Nightly clippy complained that `#[cfg(test)]` should be on the bottom of the file